### PR TITLE
Updates publications and css

### PR DESCRIPTION
**Pull request by Lin-K76**
Adds fully fleshed out publication page from 2014-2024 with consistent citation formatting. Updates css to accommodate for the very long page to have the footer not cut off anything.

### CODE REVIEW

File: data/publications.html [hunk 1]
```diff
@@ -23,17 +23,70 @@ <h1>Publications</h1>
...
+          <li><strong>Daniel Witvliet, Ben Mulcahy, James K. Mitchell, Yaron Meirovitch,  Daniel R. Berger, Douglas Holmyard, Richard L. Schalek, Steven J. Cook, Wan Xian Koh, Marianna Neubauer, Christine Rehaluk, Zitong Wang, David Kersen, Andrew D. Chisholm, Nir Shavit, Jeffrey W. Lichtman, Aravinthan Samuel, and Mei Zhen.</strong>  Invariant, stochastic, and developmentally regulated synapses constitute the C. elegans connectome from isogenic individuals. <em>Computational and Systems Neuroscience - Cosyne 2019</em> Poster Presentation, Lisbon, Portugal, March 2019.</li>
+        </ul>
         <h3>2018</h3>
+        <ul>
```
Comment (tathey1) [inline data/publications.html:66]: should this be blank?

File: data/publications.html [hunk 2]
```diff
@@ -42,4 +95,97 @@ <h2>All of Nir's Publications</h2>
         <p>&copy; 2024 Shavit Lab</p>
     </footer>
 </body>
-</html>
\ No newline at end of file
+</html>
+
+    
+    <p><strong>2018</strong></p>
```
Comment (tathey1) [inline data/publications.html]: should these entries be in the unordered list above?

**Review by tathey1 — CHANGES_REQUESTED** [review]
The list of publications looks weird - it looks like the 2014-2018 publications are not in the bulleted list but instead are at the bottom without bullets. Is this what you want?

File: data/publications.html (hunk 2 above)
Comment (Lin-K76) [reply data/publications.html]: Yeah, half of the formatting is still a WIP. I'm manually re-formatting and organizing all of the citations so it's taking a little longer than expected, so I just put a PR to mark progress and marked it as a draft.

Comment (tathey1) [reply data/publications.html]: Ok, go ahead and request my review again when you finish the changes 👍

Comment (Lin-K76) [reply data/publications.html]: Sounds good! I think it's ready now, thanks!

**Review by tathey1 — APPROVED** [review]
For good practice maybe add a description to this PR outlining what you did? :)

**Pull request merged**